03-task-manager/starter/models/Task.js

```diff
@@ -1,8 +1,16 @@
 const mongoose = require('mongoose')
 
 const TaskSchema = new mongoose.Schema({
-  name: String,
-  completed: Boolean,
+  name: {
+    type: String,
+    required: [true, 'Task name is required'],
+    trim: true,
+    maxLength: [50, 'Task name can not be more than 20 characters'],
```

Comment:
Is this intentional?  Your max length is set to 50, but your error message says 20.

Reply:
I changed the max length to 50 and forgot to update the message.